subprojects/ce-login/celogin/include/CeLogin.h

```diff
@@ -270,20 +287,31 @@ CeLoginRc checkAcfIntegrityV2(
  *  @param publicKeyLengthParm the length of the provided public key
  *  @param serialNumberParm a pointer to the serial number of the current system
  *  @param serialNumberLengthParm the length of the provided serial number
- *  @param currentReplayIdParm the current replay ID persisted by the BMC. The very first time
- *                             this function is called, the BMC may pass zero as input.
- *  @param updatedReplayIdParm the value the BMC is required to persist as the new replay ID if
- *                             the function call succeeds. This must be passed in as the
- *                             currentReplayIdParm the next time this interface is called.
+ *  @param currentReplayIdParm the current replay ID persisted by the BMC. This should always
```

Comment:
Is this better?

"the current replay ID persisted by the BMC. _Since the replay id is validated and stored at upload_, this should always be equal to the replay ID in the input ACF."